spanner_orm/tests/model_api_test.py

```diff
@@ -49,3 +50,53 @@ def test_find_result(self, find):
     self.assertEqual(result.key, 'key')
     self.assertEqual(result.value_1, 'value_1')
     self.assertIsNone(result.value_2)
+
+  @mock.patch('spanner_orm.api.SpannerApi.insert')
+  def test_create_calls_api(self, insert):
+    mock_transaction = mock.Mock()
+    models.SmallTestModel.create(mock_transaction, key='key', value_1='value')
+
+    insert.assert_called_once()
```

Comment:
this could use `assert_api_called` helper-

Reply:
that helper is just for batch_save calls, for the reason mentioned below